x-pack/plugins/triggers_actions_ui/public/types.ts

```diff
@@ -28,6 +28,7 @@ export interface ActionConnectorFieldsProps<TActionConnector> {
   editActionSecrets: (property: string, value: any) => void;
   errors: { [key: string]: string[] };
   http?: HttpSetup;
+  docLinks?: DocLinksStart;
```

Comment:
Should this be required instead of optional?
```suggestion
  docLinks: DocLinksStart;
```